### PR TITLE
Temp draft for launch

### PR DESCRIPTION
**Pull request by NicDominguez**
Brings in hotfixes since last merge and solves conflicts.

### CODE REVIEW

File: _content/_data/compliance.json [hunk 1]
```diff
@@ -909,38 +910,6 @@
               "status": "new"
             }
           ]
-        },
```
Comment (dmcalester) [inline _content/_data/compliance.json:912]: Hey Nic, looks like the Application State compliance got removed, please add this back.

File: _content/_data/meta.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
-  version: "5.2",
+  version: "5.1",
```
Comment (dmcalester) [inline _content/_data/meta.js:2]: Please bump to 5.2

**Review by dmcalester — CHANGES_REQUESTED** [review]
Couple of minor changes. Thanks Nic.

File: _content/_data/compliance.json [hunk 3]
```diff
@@ -130,6 +130,7 @@
             },
             {
               "numeral": 3,
+
```
Comment (nortonprojects) [inline _content/_data/compliance.json:133]: Same as the other review, double check your white space

**Review by nortonprojects — APPROVED** [review]

**Review by dmcalester — APPROVED** [review]

**Pull request merged**